.github/workflows/ci.yml

```
@@ -14,6 +14,9 @@ on:
     tags:
       - "*.*.*"
 
+permissions:
+  contents: read
```

```suggestion
  contents: read
  packages: read
```

AFAIU, the token needs at least `packages: read` according to https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry

@Mpdreamz can you please confirm?

---

I don't think we do, AFAIK. We only have `contents:read` over on https://github.com/elastic/elastic-otel-dotnet/blob/main/.github/workflows/ci.yml, and that is working, I believe.

---

I pushed a couple of commits to fix unrelated failures. This should be all good now!